index.html

```diff
@@ -11,33 +11,3186 @@
         shortName: "did-rubric",
         edDraftURI: "https://w3c.github.io/did-rubric/",
         wgPublicList: "public-did-wg",
+        wg: "Decentralized Identifier Working Group",
+        wgURI: "https://www.w3.org/2019/did-wg/",
         wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/117488/status",
         github:                  {
             repoURL: "https://github.com/w3c/did-rubric/",
             branch: "master"
         },
-        noRecTrack: "true"
+        noRecTrack: "true",
+        editors: [{
+          name: "Joe Andrieu"
```

Comment:
Update for Joe:
          name: "Joe Andrieu",
          company: "Invited Expert",
          url: "http://legreq.com",
          w3cid: 97261
        },{
          name: "Daniel Hardman",
          url: "https://github.com/dhh1128",
          company: "Evernym",
          companyURL: "http://evernym.com",

Reply:
I believe my w3cid = 108790. The url for me should probably be my LinkedIn profile: https://www.linkedin.com/in/danielhardman/. The other values are good.